.solhint.json

```
@@ -19,15 +19,16 @@
     "ordering": "error",
     "visibility-modifier-order": "error",
     "code-complexity": ["error", 7],
-    "function-max-lines": ["error", 50],
+    "function-max-lines": ["error", 75],
     "max-line-length": ["error", 130],
     "max-states-count": ["error", 15],
-    "no-empty-blocks": "error",
     "no-unused-vars": "error",
     "payable-fallback": "error",
     "constructor-syntax": "error",
     "not-rely-on-time": "off",
-    "reason-string": "off"
+    "reason-string": "off",
+    "no-global-import": "off",
+    "no-empty-blocks": "off"
```

```suggestion
    "no-empty-blocks": "error"
```

This should remain as an error, ignore should be put where necessary (I think you added the rule because prettier changed how the functions were written, so the current ignores didn't work anymore, the `// solhint-disable-next-line no-empty-blocks` comment only needs to be moved a bit to be effective again, I'll open a PR to your branch to fix this!).